Extensions/Physics2Behavior/physics2runtimebehavior.ts

```diff
@@ -3797,27 +3811,89 @@ namespace gdjs {
       joint.GetBodyB().SetAwake(true);
     }
 
-    // Collision
-    static collisionTest(
+    onContactBegin(otherBehavior: Physics2RuntimeBehavior) {
+      // There might be contacts that end during the frame and
+      // start again right away. It is considered a glitch
+      // and should not be detected.
+      let i = this.contactsEndedThisFrame.indexOf(otherBehavior);
+      if (i !== -1) {
+        this.contactsEndedThisFrame.splice(i, 1);
+      } else {
+        this.contactsStartedThisFrame.push(otherBehavior);
+      }
+    }
```

Comment:
I wonder if it could be interesting to test it as it needed a comment, but I don't see how to write a good test. It will be hard to cover this case with actual physics. I guess, it could be called directly by the test.